Lib/test/test_minidom.py

```diff
@@ -496,7 +511,11 @@ def testAttributeRepr(self):
         self.confirm(str(node) == repr(node))
         dom.unlink()
 
-    def testTextNodeRepr(self): pass
+    def testTextNodeRepr(self):
+        dom = Document()
+        self.addCleanup(dom.unlink)
+        text = dom.createTextNode("monty")
+        self.assertEqual(str(text), repr(text))
 
     def testWriteXML(self):
```

Comment:
```suggestion
    def testWriteXML(self):
```
@picnixz Is this want we want to go with? (You can apply it if you want)

Reply:
If we don't have a test for an `Element` (and not a `TextNode`) added via `createElement`, let's use it. Otherwise, let's remove this one. I think there is some value in testing the repr/str of an element vs a TextNode.

Reply:
There already exists a testElementReprAndStr test. I will remove this then.